pkg/builders/builders.go

```diff
@@ -43,3 +43,39 @@ func GetOwnerLabel(name, namespace, owner string) map[string]string {
 		return map[string]string{}
 	}
 }
+
+// CloneAndRemoveOwnerLabels removes the owner labels from a set of labels
+func CloneAndRemoveOwnerLabels(labels map[string]string, owner string) map[string]string {
```

Comment:
Nit: I mentioned this before in https://github.com/operator-framework/operator-marketplace/pull/223#discussion_r305064649. I don't understand why you need to write your own helper when there is  [labels.CloneAndRemoveLabel](https://godoc.org/k8s.io/kubernetes/pkg/util/labels#CloneAndRemoveLabel).

Reply:
If you end up not using this, then it should be removed.